usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/drawSphereClient.js

```diff
@@ -514,9 +512,10 @@
             Script.setTimeout(function(){
                 _this.playSound(DRAW_SOUND, DRAW_SOUND_VOLUME, lineStartPosition, true, true);
             }, SOUND_DELAY_TIME);
-
-            isTheTriggerUpdateRunning = true;
-            Script.update.connect(_this.onTriggerPressedScriptUpdate);
+            if (!_this.isTheTriggerUpdateRunning) {
+                Script.update.connect(_this.onTriggerPressedScriptUpdate);
```

Comment:
I didn't realize we were using the update loop in this script. Why does this script need to use the update loop? Why can't it use a controller mapping?